modules/registry-configuring-storage-azure-gov-cloud.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+[id="registry-configuring-storage-azure-gov-cloud_{context}"]
+= Configuring registry storage for Azure Government
+
+During installation, your cloud credentials are sufficient to create Azure Blob
+Storage, and the Registry Operator automatically configures storage.
+
+.Prerequisites
+
+* A cluster on Azure with user-provisioned infrastructure in a government region.
+* To configure registry storage for Azure, provide Registry Operator
+cloud credentials.
+* For Azure storage, the secret is expected to contain one key:
+** `REGISTRY_STORAGE_AZURE_ACCOUNTKEY`
+
+.Procedure
+
+. Create an link:https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal[Azure storage container].
+
+. Fill in the storage configuration in `configs.imageregistry.operator.openshift.io/cluster`:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry.operator.openshift.io/cluster
+----
++
+.Example configuration
+[source,yaml]
+----
+storage:
+  azure:
+    accountName: <storage-account-name>
+    container: <container-name>
+    cloudName: AzureUSGovernmentCloud <1>
+----
+<1> Defaults to `AzurePublicCloud`. You can also set `cloudname` to `AzureUSGovernmentCloud`, `AzureChinaCloud`, or `AzureGermanCloud`.
```

Comment:
Better to add the explain for cloudName, as https://github.com/openshift/api/pull/650/files#diff-3ff94e87467e8743ff79a40fa85f123d3205a560f79b4dea1919e76a4d55958dR230-R233.

Changing the cloudName need sufficient credentials. I'm afraid this description is not rigorous enough.
How about "You can also set `cloudname` to `AzureUSGovernmentCloud`, `AzureChinaCloud`, or `AzureGermanCloud` with sufficient credentials" ?